mesa/__init__.py

```diff
@@ -1,13 +1,40 @@
 '''
 Mesa Agent-Based Modeling Framework
 
-Core Objects: Model, and Agent.
+Core Objects: Agent and Model.
 '''
 
+# Import submodules
+from . import batchrunner
```

Comment:
These are import to initialize the module to trigger tests, but also is what is causing the flake8 error.